src/css/main.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["base.scss","_variables.scss"],"names":[],"mappings":"AAAA;AAOA;EACE;EACA;EACA;EACA;;;AAGF;EACE;EACA;EACA;EACA;EACA,YClBmB;EDmBnB,aCVa;;;ADqBf;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA,OCxDU;EDyDV,aCnDa;EDoDb;;;AAKF;EACE;EACA;;;AAIJ;EACE;EAEA;;;AAGF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;;AAGF;EACE;EACA;;AAGF;EACE;EACA;;;AAIJ;EACE;EACA;EACA;EACA;;;AAGF;EACE","file":"main.css"}
```

Comment:
What is this map for? I've never seen anything like it, or if I have I don't remember :p

Reply:
Whoops! I shouldn't have pushed that up. That's the file used to watch the changes I made to the SCSS and map them to CSS. ^^;